requirements/requirements-dev.in

```diff
@@ -46,5 +46,5 @@ coverage
 bump2version
 
 # Test support
-matrix-synapse==1.27.0
+matrix-synapse==1.29.0
```

Comment:
Current version is now 1.30.1
Would it be worth it to check if it's good for us and update to the latest?